_posts/2018-08-31-announcing-riff-0-1-2-on-Knative.md

```diff
@@ -0,0 +1,199 @@
+---
+layout: single
+title: "Announcing riff v0.1.2 on Knative"
+header:
+  overlay_image: /images/liverpool-st.jpg
+excerpt:
+  With install manifests, verbose builds, and function chaining using channels and subscriptions
+permalink: /blog/announcing-riff-0-1-2-on-Knative/
+---
+
+[riff v0.1.2](https://github.com/projectriff/riff/releases/tag/v0.1.2) on Knative is now available.  
+Many thanks to all the riff and Knative contributors.
+
+## install
+We recommend installing riff v0.1.2 on a fresh Kubernetes cluster. The riff CLI can be downloaded from our [releases page](https://github.com/projectriff/riff/releases/tag/v0.1.2) on GitHub. Please follow one of the [getting started](/docs) guides, to create a new cluster on GKE or minikube. Remember that you can also use the CLI to uninstall everything.
+
+#### uninstall knative and istio without prompting
+```sh
+riff system uninstall --istio --force
+```
+
+Riff will install "stable" release builds of Knative serving, eventing, and build components. You can opt to install the latest nightly builds using the new `--manifest latest` option.
+
+#### install latest build on minikube (for GKE omit --node-port)
+```sh
+riff system install --manifest latest --node-port
```

Comment:
do we really want users to install the latest? I would suggest leaving the `--manifest` part out here and mention that it can be used to install using either latest or a custom template file